src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Main.cs

```diff
@@ -76,11 +81,9 @@ public List<Result> Query(Query query)
 
                                 hide = true;
                             }
-                            catch (Win32Exception)
+                            catch (Win32Exception ex)
                             {
-                                var name = $"Plugin: {_context.CurrentPluginMetadata.Name}";
-                                var msg = "Can't Open this file";
-                                _context.API.ShowMsg(name, msg, string.Empty);
```

Comment:
Why removing existing functionality? Please add message box again.

Reply:
![image](https://github.com/user-attachments/assets/d14e725e-433f-45d1-ac0f-8dfd3355729b)

Reply:
Technically this works. But logging and showing all message are different things. Either you rename the method or you split the methods.